backend/src/Prelude/Json.fs

```diff
@@ -67,6 +67,28 @@ module Vanilla =
       else
         writer.WriteNumberValue(value)
 
+
+  type Int128Converter() =
```

Comment:
I think you're right to have these as strings rather than as RawValues, but can you add a comment explaining?

Reply:
I don't think there's an option to use RawValues here, as `reader` and `writer` don't  have a `GetRawValue()` function.